test/coverlet.core.tests/Coverage/CoverageTests.cs

```diff
@@ -10,8 +10,8 @@ namespace Coverlet.Core.Tests
 {
     public partial class CoverageTests
     {
-        private readonly InstrumentationHelper _instrumentationHelper = new InstrumentationHelper(new ProcessExitHandler(), new RetryHelper(), new FileSystem());
-        private readonly Mock<ILogger> _mockLogger = new Mock<ILogger>();
+        private static readonly Mock<ILogger> _mockLogger = new Mock<ILogger>();
```

Comment:
keep instance semantic...it's not a great problem for test this instances(no static)

Reply:
Done